pylintrc

```diff
@@ -301,6 +301,9 @@ ignored-modules=typed_ast.ast3
 # (useful for classes with attributes dynamically set).
 ignored-classes=SQLObject
 
+# Regex pattern to define which classes are considered mixins.
+mixin-class-rgx=.*Mix[i|I]n
```

Comment:
@Pierre-Sassoulas This will also match `Mix|n`. It should just be `.*Mix[Ii]n`.

Reply:
😄 Obviously I'm just handing the case with really big using pipe, and did not make a regex mistake at all.